qemu/tests/nic_hotplug.py

```diff
@@ -140,12 +140,21 @@ def run_sub_test(params, plug_tag):
                  if needed
         """
         sub_type = params.get("sub_type_%s" % plug_tag)
-        if sub_type:
-            context_msg = "Running sub test '%s' %s"
-            error_context.context(context_msg % (sub_type, plug_tag),
-                                  logging.info)
-            utils_test.run_virt_sub_test(test, params, env, sub_type)
-            return (sub_type == "shutdown")
+        login_timeout = params.get_numeric("login_timeout", 360)
+        session = vm.wait_for_serial_login(timeout=login_timeout)
+        shutdown_method = params.get("shutdown_method", "shell")
+        shutdown_command = params.get("shutdown_command")
+        if sub_type == "boot":
```

Comment:
I am not sure if rename it to `reboot` would be better, it's up to you.

Reply:
The comments above have been resolved, please help me review it again. Thanks a lot.